src/target/riscv/riscv-013.c

```diff
@@ -336,11 +336,36 @@ static unsigned int decode_dm(char *text, unsigned int address, unsigned int dat
 		unsigned int address;
 		enum riscv_debug_reg_ordinal ordinal;
 	} description[] = {
-		{DM_DMCONTROL, DM_DMCONTROL_ORDINAL},
 		{DM_DMSTATUS, DM_DMSTATUS_ORDINAL},
+		{DM_DMCONTROL, DM_DMCONTROL_ORDINAL},
+		{DM_HARTINFO, DM_HARTINFO_ORDINAL},
+		{DM_HAWINDOWSEL, DM_HAWINDOWSEL_ORDINAL},
+		{DM_HAWINDOW, DM_HAWINDOW_ORDINAL},
 		{DM_ABSTRACTCS, DM_ABSTRACTCS_ORDINAL},
 		{DM_COMMAND, DM_COMMAND_ORDINAL},
-		{DM_SBCS, DM_SBCS_ORDINAL}
+		{DM_ABSTRACTAUTO, DM_ABSTRACTAUTO_ORDINAL},
+		{DM_CONFSTRPTR0, DM_CONFSTRPTR0_ORDINAL},
+		{DM_CONFSTRPTR1, DM_CONFSTRPTR1_ORDINAL},
+		{DM_CONFSTRPTR2, DM_CONFSTRPTR2_ORDINAL},
+		{DM_CONFSTRPTR3, DM_CONFSTRPTR3_ORDINAL},
+		{DM_NEXTDM, DM_NEXTDM_ORDINAL},
+		{DM_DATA0, DM_DATA0_ORDINAL},
+		{DM_PROGBUF0, DM_PROGBUF0_ORDINAL},
+		{DM_AUTHDATA, DM_AUTHDATA_ORDINAL},
+		{DM_DMCS2, DM_DMCS2_ORDINAL},
+		{DM_HALTSUM0, DM_HALTSUM0_ORDINAL},
+		{DM_HALTSUM1, DM_HALTSUM1_ORDINAL},
+		{DM_HALTSUM2, DM_HALTSUM2_ORDINAL},
```

Comment:
Is this oversight from another change? This doesn't seem to have anything to do with the description of this PR. Can you update the description to mention every significant change from this PR?

Reply:
Thanks! Removed the change to `decode_dm()`. Commit description is updated to mention the addition of `<no decoding available>`.